nodebalancer.go

```diff
@@ -32,6 +32,9 @@ type NodeBalancer struct {
 	// An array of tags applied to this object. Tags are for organizational purposes only.
 	Tags []string `json:"tags"`
 
+	// The ID of an associated firewall with this NodeBalancer.
+	FirewallID int `json:"firewall_id"`
+
```

Comment:
I think if the returning API object doesn't include the firewall ID, we shouldn't put it in the NodeBalancer struct because some functions like `GetNodeBalancer` wouldn't know the firewall ID and will leave this field empty (value `0`). I am not 100% confident about this though, and I wonder how we handle a similar situation for other existing structs.

Reply:
We can wait on confirmation from the API team maybe? These changes can't be merged into main either way so I don't think it would hurt to follow-up.

Reply:
Yea, but they confirmed we won't have that in the response...